docs/developer-docs/latest/developer-resources/data-management.md

```diff
@@ -146,15 +140,77 @@ npm strapi export --no-compress
 </code-block>
 </code-group>
 
+### Export only selected types of data
+
+The default `strapi export` command exports your content (entities and links), files (assets), project configuration, and schemas. The `--only` option allows you to export only the listed items by passing a comma-separated string. The available values are `content`, `files`, and `config`. Schemas are always exported, as schema matching is used for `strapi import`.
```

Comment:
It might be worth specifying with "files" that media entities in the database will still be there, but we will not copy the actual files over.

Reply:
The way I interpret this is that that if you exclude assets the file is not imported (or exported) but the database record is still imported (exported). I added a callout to explain this.